implants/eldritch/src/pivot/port_scan_impl.rs

```diff
@@ -615,17 +587,8 @@ mod tests {
     // verify our non async call works and Dict return type.
     #[test]
     fn test_portscan_return_type_starlark_dict_from_interpreter() -> anyhow::Result<()>{
-        // Setup test ports
-        let runtime = tokio::runtime::Builder::new_current_thread()
-            .enable_all()
-            .build()
-            .unwrap();
-
-        let response = runtime.block_on(
-            allocate_localhost_unused_ports(4,"tcp".to_string())
-        );
 
-        let test_ports = response.unwrap();
+        let test_ports: Vec<i32> = (8000..8004).map(|x| x).collect();
```

Comment:
nit: Might be more readable to just list out the 4 values?

Reply:
I'm so cool though.